.circleci/get_data.sh

```diff
@@ -1,26 +1,36 @@
 
-if [[ "$SHELL" =~ zsh ]]; then
+if [[ "$SHELL" == zsh ]]; then
   setopt SH_WORD_SPLIT
 fi
 
 # Edit these for project-wide testing
 WGET="wget --retry-connrefused --waitretry=5 --read-timeout=20 --timeout=15 -t 0 -q"
-LOCAL_PATCH= ~/projects/xcp_d/xcp_d  # Change to your local patch 
-IMAGE=pennlinc/xcp_d:unstable
+LOCAL_PATCH="~/projects/xcp_d/xcp_d"  # Change to your local patch
```

Comment:
I noticed that the space was causing problems sometimes. Valid paths weren't being found.

Reply:
great catch